components/encumbered/audacity/patches/16-fix-unknown-method.patch

```diff
@@ -0,0 +1,19 @@
+Rude fix for the following compilation error
```

Comment:
All over the code NoteTrack is often treated as PlayableTrack, however earlier it could be based either on PlayableTrack or AudioTrack based on EXPERIMENTAL_MIDI_OUT.